### ISSUE TITLE
Confusuion because two repos exists which are basically the same

### ISSUE THREAD

**Issue by metashock** [pycontribs/jira#77]
Hi,

the fact that there is another repo https://bitbucket.org/bspeakmon/jira-python/overview which seems the origin of this repo, **plus** there is even a separate PyPi package for version 0.31 (called python-jira) built out of that repo is really confusing!

Why don't you mentioned in the README that you forked that project? Why don't you (or bspeakmon) remove the outdated package from PyPi? Why didn't you just contributed to the existing repo?

Thorsten

**Comment by Hillgod** [comment]
It's incredibly confusing, but it does seem that the BitBucket repo is the one with active development. There are bugs in this one that appear to have been fixed. It's unfortunate that this appears to be what's published on PyPi.

It'd be nice if Atlassian would endorse clients and put them on their site. Searching 'python jira api' provides no clear answer on what library to use.

**Comment by metashock** [comment]
@Hillgod No, this repo is the one with active development. Both share the same history until commit 6897f63cf459eeb98c80638273ffa3ba8f99bd70 (including it).. After that no changes have been introduced by the bitbucket repo while many changes were applied to *this* github repo.

So I suggest to use this one. I admit that I was really confused, I would add an explanation to the README.md

**Comment by awurster** [comment]
hi guys. Andrew from Atlassian here. i am not an owner - more of a user of this package.  and whil i'm not officially making a response on behalf of Atlassian - ill try to help wherever i can.

we could for instance "delete" the repo and redirect it to the new github project page if you prefer. 
@ssbarnea ? @bspeakmon ?

let me know...

**Comment by metashock** [comment]
@awurste Hey Andrew! I would really appreciate that! The current situation is pretty unsatisfying.  Looks like we need to wait what the repo owners will say! :) However, I really wonder how that guys could run into that situation...

**Comment by bspeakmon** [comment]
I'll add a note to my repo saying that the action is over here.

**Comment by metashock** [comment]
@bspeakmon What means "The action is over here"? Can you add a few more sentences?

**Comment by bspeakmon** [comment]
* The bitbucket repo is my personal repo where I started the project in 2012 while still at Atlassian.
* About a year back Sorin asked to move the project over here (to github) as part of pycontribs. I agreed.
* You are confused as to which one is "active" or "official". The one here on github is "active" and "official".
* I have added a note on the README in my repo pointing people to this github repo for active development to help people like you who were confused.
* I am not going to delete my personal repo from bitbucket.
